cpp/csp/python/PyStructFastList_impl.h

```diff
@@ -511,6 +511,27 @@ static PyObject * PyStructFastList_Reversed( PyStructFastList<StorageT> * self,
     CSP_RETURN_NULL;
 }
 
+template<typename StorageT>
+static PyObject * PyStructFastList_reduce( PyStructFastList<StorageT> * self, PyObject * Py_UNUSED( ignored) )
+{
+    CSP_BEGIN_METHOD;
+    
+    typename VectorWrapper<StorageT>::IndexType sz = self -> vector.size();
+    PyObject * args = PyTuple_New( sz );
+    PyObjectPtr list = PyObjectPtr::own( toPython( self -> vector.getVector(), self -> arrayType ) );
```

Comment:
since we are already converting to list there is really no point to convert to tuple as well
Try to just return
Py_BuildValue( "O(O)", &PyList_Type, list);

Reply:
Done.